selftests/functional/test_loader.py

```diff
@@ -350,6 +350,19 @@ def test_python_unittest(self):
         self.assertFalse(exps, "Some expected result not matched to actual"
                          "results:\n%s\n\nexps = %s" % (result, exps))
 
+    def test_list_subtests_filter(self):
+        """
+        Check whether the subtests filter works for both INSTRUMENTED
+        and SIMPLE in a directory list.
+        """
+        cmd = "%s list examples/tests/:fail" % AVOCADO
+        result = process.run(cmd, ignore_status=True)
```

Comment:
I don't see the reason for `ignore_status=True` here.  If avocado spits out a non-zero status, we should take notice.

Reply:
Yeah, but think about it: in theory the filter could generate some kind of unrelated error and still generate the expected output.  Better to be safe, as it costs nothing.

Reply:
Sure thing.